### PR TITLE
In Do_Compilation_Unit the return value Unit_Symbol was not set for a…

### PR DESCRIPTION
**Pull request by tjj2017**
… package body

To determine a symbol value for a package body a symbol is determined from
its specification and registered in the symbol table.  This required checking for
the package being a child.
Note: The same check may be required for child subprograms!

### CODE REVIEW

File: gnat2goto/driver/tree_walk.adb [hunk 1]
```diff
@@ -1,4 +1,3 @@
-with Uname;                 use Uname;
```
Comment (xbauch) [inline gnat2goto/driver/tree_walk.adb]: You may have wanted to remove this after the second commit.

**Comment by xbauch** [comment]
@tjj2017 I have un-removed the `uname` include and updated the golden results into #231.

**Comment by chrisr-diffblue** [comment]
@tjj2017 - Could you do a git rebase rather than a merge for this PR as well, thanks :-)

**Comment by tjj2017** [comment]
Superseded by PR #231

**Pull request closed without merging**